src/main/resources/templates/subscription.html

```diff
@@ -169,10 +170,38 @@ <h3 class="modal-title text-center">Subscription Form</h3>
                                         class="glyphicon glyphicon-plus float-right"></i> Add Key/Value Pair
                                 </button>
                             </div>
-
+                        </div>                        
+                        
+                        <div class="form-group">
```

Comment:
add here class pt-3 for example (like div class="pt-3 form-group"). It will add little bit more space between this and previous blocks and 'Authorization' labe; will not shift.